nativelib/src/main/scala/scala/scalanative/native/time.scala

```diff
@@ -0,0 +1,84 @@
+package scala.scalanative
+package native
```

Comment:
`scalanative.native` may only contain bindings to C's standard library definitions. This one seems to be a binding for POSIX time, therefore should go to `scalanative.posix` instead.